common-lib/project.clj

```diff
@@ -65,6 +65,7 @@
   :profiles {:security {:plugins [[com.livingsocial/lein-dependency-check "1.4.1"]]
                         :dependency-check {:output-format [:all]
                                            :suppression-file "resources/security/suppression.xml"}}
+             ;; :uberjar {:aot :all} ;; remove before merging
```

Comment:
Just putting a comment here to remind you to do this before merging

Reply:
thanks, I actually have a couple of others too.